src/mgl-pax.el

```diff
@@ -129,6 +129,8 @@
   Lisp system.")
 
 
+(defvar mgl-pax-loaded-backends nil)
+
 (defvar mgl-pax-edit-lisp-definition-hook nil)
```

Comment:
Why not `mgl-pax-edit-definition-hook` to directly parallel `slime-edit-definition-hook`?

Reply:
I'm not too happy with the name either, but I think `mgl-pax-edit-definition-hook` might give out the impression that it gets called by `mgl-pax-edit-definition` (while it's actually the opposite!), no?

Reply:
OK.